src/common/options/mds.yaml.in

```diff
@@ -347,22 +347,22 @@ options:
 - name: mds_session_cap_acquisition_throttle
   type: uint
   level: advanced
-  desc: throttle point for cap acquisition decay counter
+  desc: Threshold at which the cap acquisition decay counter throttles
```

Comment:
convention is that "desc" is not a sentence and the first word is not capitalized.

Reply:
ok, I can change that. But this gets imported to documentation where it becomes incorrect. Is that fine?

Reply:
@kotreshhr, @batrick: I'll do my best to institutionalize this convention of making "desc"s neither sentences nor capitalized.